accounts/admin.py

```diff
@@ -1 +1,6 @@
-# from django.contrib import admin
+from django.contrib import admin
+from django.contrib.auth import get_user_model
+
+CustomUser = get_user_model()
+
+admin.site.register(CustomUser)

```

Comment:
```suggestion
User = get_user_model()

admin.site.register(User)
```
forms.pyやtests.pyで命名しているように、変数名はUserでOKです
統一感をもたせることが(同じ意味を持つものは同じ名前での宣言が)望ましいと思います。